pkgs/games/crossfire/crossfire-arch.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, lib, fetchsvn,
+  version, rev, sha256 }:
+
+stdenv.mkDerivation rec {
+  version = "r${toString rev}";
+  name = "crossfire-arch-${version}";
+
+  src = fetchsvn {
+    url = "http://svn.code.sf.net/p/crossfire/code/arch/trunk/";
+    sha256 = sha256;
+    rev = rev;
+  };
+
+  hydraPlatforms = [];
+  phases = ["unpackPhase" "installPhase"];
+  installPhase = ''
+    mkdir -p "$out"
+    cp -a . "$out/"
+  '';
+
+  meta = with lib; {
+    description = "Archetype data for the Crossfire free MMORPG";
+    homepage = "http://crossfire.real-time.com/";
+    license = licenses.gpl2;
```

Comment:
Do you know if the license is gpl2Only or gpl2Plus?

Reply:
Looks like gpl2Plus, based on the source headers.